pkg/kubelet/network/cni/cni.go

```diff
@@ -271,7 +272,11 @@ func (network *cniNetwork) deleteFromNetwork(podName string, podNamespace string
 	err = cninet.DelNetwork(netconf, rt)
 	if err != nil {
 		glog.Errorf("Error deleting network: %v", err)
-		return err
+
+		subStr := "no such file or directory"
```

Comment:
This doesn't seem like robust error-checking, can't https://godoc.org/os#IsNotExist be used here?